corehq/apps/reports/standard/monitoring.py

```diff
@@ -264,10 +264,17 @@ def totals_columns(self):
 
     @property
     def special_notice(self):
-        if self.domain_object.case_sharing_included():
-            return _("This report currently does not support case sharing. "
-                     "There might be inconsistencies in case totals if the "
-                     "user is part of a case sharing group.")
+        if self.has_case_sharing:
+            help_link = (
+                "https://dimagi.atlassian.net/wiki/spaces/commcarepublic"
+                "/pages/2143957523/Debugging+Case+Sharing+errors"
+            )
+            return format_html(_(
+                "Note that when viewing this report by group it will only include "
+                "cases which are assigned to a Case Sharing Group/Location. Learn "
+                "more about troubleshooting issues with Case Sharing Groups "
```

Comment:
troubleshooting? Confused by that term in a report.
Why are we adding it here?

Reply:
I pulled this text from the acceptance criteria of the linked Jira ticket, but you make a good point. There's nothing that went wrong, so I can see it being confusing that we're linking to troubleshooting documentation.

I had an offline discussion with Mary regarding the above text and will be changing it to:
> Note that when viewing this report by group it will only include cases which are assigned to a Case Sharing Group/Location. To learn more about Case Sharing click on [this](https://dimagi.atlassian.net/wiki/spaces/commcarepublic/pages/2143957537/Case+Sharing) documentation.

The above change serves to link the user to more general documentation if they'd like to learn more about Case Sharing.

Reply:
[e7cae05](https://github.com/dimagi/commcare-hq/pull/34464/commits/e7cae057ea4a890bf8e9df8ba68eda02dccd4c4a)